libs/coin-modules/coin-aptos/src/constants.ts

```diff
@@ -45,6 +45,10 @@ export enum DIRECTION {
   UNKNOWN = "UNKNOWN",
 }
 
-export const SUPPORTED_TOKEN_TYPES = ["coin", "fungible_asset"];
+export enum TOKEN_TYPE {
+  COIN = "coin",
+  FUNGIBLE_ASSET = "fungible_asset",
+}
+
```

Comment:
you don't need to change this, have a look at the `buildTransaction`

assume

```ts
export type AptosTokenInfo = {
  tokenType: string;
  contractAddress: string;
};
```

As I am doing in the PR for estimatefees